src/hooks/useScrollToTop.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react';
+import { useLocation } from 'react-router';
+import { useNote } from '../context';
+
+export function useScrollToTop() {
```

Comment:
Appreciation: Good that you've implemented this hook.

Reply:
Thanks